### PR TITLE
Cards Vue M

### PR DESCRIPTION
**Pull request by sienmich**
My first commit here. I'll be grateful If you could tell me if I'm doing the right thing. Thanks!

### CODE REVIEW

**Review by chosta — APPROVED** [review]
Great work!
I think putting description on the bottom of the metadata interface makes more sense. I'll start doing it myself.
Feel free to work on the cards. You can skip the ones you are unsure about and tag me.

File: src/styles/cards_v2.less [hunk 1]
```diff
@@ -337,8 +337,6 @@
             }
             .card-tag-science {
                 background-image: url("./assets/tags/science.png");
-                background-size: 46px;
```
Comment (kberg) [inline src/styles/cards_v2.less:340]: Just making sure this is intentional.

**Comment by kberg** [comment]
> Great work!
> I think putting description at the bottom of the metadata interface makes more sense. I'll start doing it myself.
> Feel free to work on the cards. You can skip the ones you are unsure about and tag me.

I agree. This is terrific. Thanks for jumping in!

File: src/styles/cards_v2.less (hunk 1 above)
Comment (chosta) [reply src/styles/cards_v2.less:340]: Yeah, a bit funny but correct I think. I intend to add some variables and mixins to the less file once we have all card items described. I wish we had js-in-css :)

Comment (sienmich) [reply src/styles/cards_v2.less:340]: All other card tags don't have `background-size` nor `background-position`, so I think the science tag also shouldn't. After this change the Mars Uni looks good.

File: src/components/card/CardRenderItemComponent.ts [hunk 2]
```diff
@@ -33,6 +33,12 @@ export const CardRenderItemComponent = Vue.component('CardRenderItemComponent',
         if (this.item.size !== undefined) {
           classes.push(`card-ocean--${this.item.size}`);
         }
+      } else if (type === CardRenderItemType.GREENERY) {
+        classes.push('card-global-requirement');
+        classes.push('card-greenery-global-requirement');
+        if (this.item.size !== undefined) {
+          classes.push(`card-greenery--${this.item.size}`);
+        }
```
Comment (sienmich) [inline src/components/card/CardRenderItemComponent.ts:41]: I'm not sure about this one though. I copied it from above (oceans). But the greeneries are not really a `card-global-requirement`. Any suggestions?

Comment (chosta) [reply src/components/card/CardRenderItemComponent.ts:41]: They can be the same as `CardRenderer.city()`. It's not super important as long as the css is correct but for the sake of consistancy, they can be a `card-tile` with `card-greenery--${size}`

**Comment by chosta** [comment]
@sienmich Let's remove the "Draft" and merge this. I need the greenery icon elsewhere :)

**Comment by sienmich** [comment]
@chosta Ok, go ahead :)

**Pull request merged**